src/components/structures/RoomDirectory.tsx

```diff
@@ -322,9 +323,8 @@ export default class RoomDirectory extends React.Component<IProps, IState> {
 
     private onFilterChange = (alias: string) => {
         this.setState({
-            filterString: alias?.trim() || "",
+            filterString: transformSearchTerm(alias?.trim()) || "",
         });
-
```

Comment:
please don't remove blank lines needlessly - they are an important part of code clarity